designs/dragonphy_top/qtm/analog_core.qtm.tcl

```diff
@@ -312,6 +312,20 @@ set_qtm_port_drive -type qtm_drive { adbg_intf_i_inbuf_out_meas }
 create_qtm_port -type input { adbg_intf_i_en_TDC_phase_reverse }
 set_qtm_port_load -type qtm_load -factor 2 { adbg_intf_i_en_TDC_phase_reverse }
 
+# ADC retimer (input to analog core but appears at the bottom of acore_debug_intf)
```

Comment:
the retimer mux control signals are now on the analog debug interface, not the digital debug interface